sample-status-updater-domain/src/main/groovy/life.qbic.samplestatusupdater.search/SampleRegistration.groovy

```diff
@@ -5,12 +5,12 @@ import groovy.transform.EqualsAndHashCode
 import java.time.Instant
 
 @EqualsAndHashCode
-class SampleModification {
+class SampleRegistration {
 
   final String sampleCode
   final Instant modificationTime
 
-  SampleModification(String sampleCode, Instant modificationTime) {
+  SampleRegistration(String sampleCode, Instant modificationTime) {
```

Comment:
to be consistent with the method name i suggest the following:

```suggestion
  SampleRegistration(String sampleCode, Instant registration) {
```

NIT: I try to not include the type in the variable name to keep it concise. Here the type is expressive by the `Instant` imo. For String that is not the case, including a sample code class would be the expressive solution.